_store/fpga.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: store-products
+
+theme: theme-yellow
+
+title: Nengo FPGA Licenses
+tagline: NengoFPGA licenses for various FPGA boards
+
+categories:
+  - fpga
+  - summer-school
```

Comment:
Should this be in `summer-school.md`?

Reply:
Yeah, this is no longer needed. I will remove it.